app/assets/javascripts/reducers/projectBoard.js

```diff
@@ -24,6 +28,16 @@ const projectBoardReducer = (state = initialState, action) => {
       ...state,
       error: action.error
     };
+  case actionTypes.SEARCH_STORIES_SUCCESS:
+    return {
+      ...state,
+      keyWord: action.keyWord,
```

Comment:
Should this be inside a `search` attribute? `storyBoard.keyword` doesn't have a lot of meaning. It could be `storyboard.search.keyword`